src/bootlint.js

```diff
@@ -937,12 +959,7 @@ var LocationIndex = _location.LocationIndex;
             var elem = $(this);
             var urlAttr = (tagNameOf(this) === 'LINK') ? 'href' : 'src';
             var pathSegments = parseUrl(elem.attr(urlAttr)).pathname.split('/');
```

Comment:
OK, after including this I end up with this function:

```javascript
/**
  * This function returns the version of an element with the element it was found on.
  * @returns {string, object}
  */
function getSemverFromURL($, object) {
        var elem = $(object);
        var urlAttr = (tagNameOf(object) === 'LINK') ? 'href' : 'src';
        var pathSegments = parseUrl(elem.attr(urlAttr)).pathname.split('/');
        var matches = pathSegments.map(function (segment) {
            var match = segment.match(/^\d+\.\d+\.\d+$/);
            return match ? match[0] : null;
        }).filter(function (match) {
            return match !== null;
        });
        if (!matches.length) {
            return;
        }
        var version = matches[matches.length - 1];

        return {
            elem: elem,
            version: version
        };
    }
```

But get the error:
```shell
200:8  error  Expected no return value  consistent-return
```

I've tried it with and without the comment block to try to address this. Sorry again for not getting this, I know it's probably something small and simple.